src/CopyOnWrite/build/Microsoft.Build.CopyOnWrite.targets

```diff
@@ -0,0 +1,6 @@
+﻿<Project>
+    <UsingTask
```

Comment:
Have you tested this?  I helped Erik a while back and we discovered that for `<UsingTask />` declaration, the first one wins.  Maybe this is still first because the `Copy` task comes from the `.tasks` file in MSBuild?  I'm not sure the import order for that one.

I think it would still be safer for this to be in the `.props`.

Reply:
I tested this as both an SDK or PackageReference.

https://github.com/dotnet/msbuild/blob/446f42e6d1478acb44faec787f90f57f38c08b73/src/Build/BackEnd/TaskExecutionHost/TaskExecutionHost.cs#L811

Based on that, declared anywhere in the Project wins over the *.tasks file in the MSBuild directory. This feels like more of a `.targets` thing, and I already tested that way. But I don't feel strongly about it.